### PR TITLE
Implemented CodeSnippetModule

### PR DESCRIPTION
**Pull request by blaumeise20**
This module allows you to save and show code snippets.

### CODE REVIEW

File: DiscordBot.Modules/CommandModules/CodeSnippetModule.cs [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace DiscordBot.Modules.CommandModules
+{
+    public class SnippetInfo
```
Comment (JKamsker) [inline DiscordBot.Modules/CommandModules/CodeSnippetModule.cs:18]: Separate class pls

Comment (blaumeise20) [reply DiscordBot.Modules/CommandModules/CodeSnippetModule.cs:18]: What do you mean? This is a seperate class! Or should I put it into a seperate file?

Comment (JKamsker) [reply DiscordBot.Modules/CommandModules/CodeSnippetModule.cs:18]: Yea i meant file 😅

Comment (blaumeise20) [reply DiscordBot.Modules/CommandModules/CodeSnippetModule.cs:18]: Where should I put it? In DiscordBot.Domain/Abstractions ? And what should I add to the Program.cs ?

**Pull request merged**